app/assets/stylesheets/survey_trip_stops.scss

```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the SurveyTripStops controller here.
+// They will automatically be included in application.css.
```

Comment:
These files can also be safely deleted, we won't need separate sass files in such a small project.